tests/test_downloader.py

```diff
@@ -157,23 +157,4 @@ def side_effect(*args, **kwargs):
             assert (args[0] == call[1])
 
         mock_get_api_data.side_effect = side_effect
-        downloader.get_recording_data(call[0])
-
-
-def test_retrieve_url_from_play_page(mocker):
-
-    with requests_mock.mock() as req_mock:
-        req_mock.get(requests_mock.ANY, status_code=200,
-                     content='<source src="https://ssrweb.zoom.us/cmr/replay" type="video/mp4" />'.encode('utf-8'))
-        resp = downloader.retrieve_url_from_play_page("https://www.example.com")
-        assert (resp == "https://ssrweb.zoom.us/cmr/replay")
-
-        rec_data = [('no videos here', 'No source element found'),
-                    ('<div id=password_form> </div>', 'Password protected')]
-
-        for content, message in rec_data:
-            with pytest.raises(downloader.PermanentDownloadError) as excinfo:
-                req_mock.get(requests_mock.ANY, status_code=200,
-                             content=content.encode('utf-8'))
-                downloader.retrieve_url_from_play_page("https://www.example.com")
-            assert(message in str(excinfo.value))
\ No newline at end of file
+        downloader.get_recording_data(call[0])
```

Comment:
mostly aesthetic, but you should see if your editor has an option to automatically insert a newline at the end of every file